app/models/film.rb

```diff
@@ -0,0 +1,23 @@
+class Film < ActiveRecord::Base
+has_many :categories_films
+has_many :categories, through: :categories_films
+has_many :reviews
+
+has_many :ratings, as: :rateable
+has_many :comments, as: :commentable
+
+validates :title,:description, presence: true
+
+  def total_ratings
+    if ratings.any?
```

Comment:
You shouldn't need the if `ratings.sum(:rating)` should return 0 if there aren't any. You're returning nil, which may not be what you want.